### ISSUE TITLE
Impact of label smoothening on librispeech

### ISSUE THREAD

**Issue by shubham-IISc** [espnet/espnet#264]
I was trying experimenting with unigram label smoothing for librispeech dataset. I tried with different lsm weights of 0.01,0.02,0.05, but results are always worse than the case of no label smoothing(ON TEST_CLEAN)
Why overconfidence does not affect the model performance in this case? Is this because of clean test data? Kindly help.

**Comment by sw005320** [comment]
I haven't tried the label smoothing in this data. @takaaki-hori, do you have comments about it?

**Comment by takaaki-hori** [comment]
Me, either. I just checked error reduction on WSJ and Voxforge recipes.
Did you train the model with the full training set (train_960)?
With a large amount of training data for a small set of labels, the smoothing might have no effect.
But I have no idea about the reason at this moment. If you train a model with a small subset of Librispeech, we maybe figure it out.

**Comment by shubham-IISc** [comment]
I have trained using train clean 360hrs data and test on test clean 100hrs data.  Can you please confirm if label smoothing improved results for  (CLEAN) test data of some other dataset.?  
 It seems to me that overconfidence is not harming on clean test data because label smoothing allows the model to indicate sensible alternatives to a given character(such as its homophones) and data being clean, generating alternatives might not be useful as peaked probability prediction would have been correct(because of no noise). I might be wrong. Kindly share your opinion on the same.

**Comment by shubham-IISc** [comment]
or maybe 360hrs is still huge training data as compared to WSJ training data.  So, maybe because of this reason , smoothing is not having any impact in this case.
